packages/@o3r/test-helpers/src/utilities/package-manager.ts

```diff
@@ -231,6 +262,23 @@ export function setPackagerManagerConfig(options: PackageManagerConfig, execAppO
       execFileSync('yarn', ['config', 'set', 'unsafeHttpWhitelist', '127.0.0.1'], execOptions);
       break;
     }
+    case 'yarn1': {
+      if (options.yarnVersion) {
+        execFileSync('yarn', ['set', 'version', options.yarnVersion], execOptions);
+      }
+      const ignoreRootCheckConfig = '--add.ignore-workspace-root-check';
+      const yarnRcPath = join(execOptions.cwd as string, '.yarnrc');
+
+      if (existsSync(yarnRcPath)) {
```

Comment:
why not using a child process to add this config ?

Reply:
because of an issue with yarn 1 https://github.com/yarnpkg/yarn/issues/6499